ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test/test_systemd.t

```diff
@@ -22,6 +22,7 @@
   $ sleep 1
```

Comment:
This needs to go

Reply:
To be clear, the wait is for the background (server) task because `/test_systemd.exe --notify` should be synchronous unless it spawns a background process.